modules/assignment/src/main/java/com/intuit/wasabi/assignment/impl/AssignmentsImpl.java

```diff
@@ -497,12 +486,12 @@ public Assignment putAssignment(User.ID userID, Application.Name applicationName
                     }
 
                     // Add the assignment to the global list of userAssignments of the user
-                    if (assignment.getStatus() != Assignment.Status.EXPERIMENT_EXPIRED) {
+                    if (Experiment.State.PAUSED != experiment.getState()) { // the Experiment is set to PAUSED in the case of a reached endDate
```

Comment:
@AndreaSuckro @calvinwma more description:  As per current business logic, user is attempted to assigned to the higher priority experiment, even if that higher priority experiment is PAUSED (assignment could be EXISTING_ASSIGNMENT or EXPERIMENT_PAUSED), user should NOT be attempted to be assigned to any other lower priority experiment...

This business rule will break if above  <IF> condition is added...